### DISCUSSION TITLE
ArgoCD Metrics; Target Revision

### DISCUSSION THREAD

**Discussion by aliasboink** [argoproj/argo-helm#2863] [General]
Hello, 

I am reading through: https://argo-cd.readthedocs.io/en/stable/operator-manual/metrics/ and I cannot see any metrics that would showcase the `targetRevision`. I've enabled this on a cluster and I'm looking through the metrics; none which seem to have the targetRevision.

I would like to have this metric to be able to create a table with each application the ArgoCD deployment has alongside its version (in this case being a GH tag under "targetRevision") to know which need to be updated and so on; general information gathering. 

Is there anything I'm missing or is there any particular reason why this data is not persent?
Thank you!

**Comment by yu-croco** [comment]
Hi @aliasboink , since it's related to Argo CD itself, so I think you can get more help in upstream?
